conf/messages-notification-codes-cds.en

```diff
@@ -21,7 +21,8 @@ decoder.soe.Frustrated = Frustrated
 
 decoder.soe.ConsolidationOpen = The MUCR is still open, or contains no declarations.
 decoder.soe.ConsolidationClosedWithoutP2P = Consolidation is closed but does not have P2P
-decoder.soe.ConsolidationHasP2P = Consolidation has P2P
+decoder.soe.ConsolidationHasP2P = Consolidation has permission to proceed
+decoder.soe.ConsolidationWithEmptyMucr = Consolidation With Empty MUCR
```

Comment:
Are you sure about all the Capital letters? It looks weird to have "... With ...".

Reply:
Crap... you are right.... that is my fault... Adding content during a call does not work as well as I would have thought